tokenizer/rainscriptTokenizerRules.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * The tokenizer rules, ordered by priority. All tokenizer rules go here.
+ *
+ * tokenizerRules:
+ *   {
+ *     pattern: RegExp,
+ *     type: any,
+ *     start: Array<int>?
```

Comment:
Please also explain the purpose of pattern, type, and start.

Reply:
Honestly, I'm not really sure what start does yet. It's part of the [lex](https://www.npmjs.com/package/lex) node package, supposedly an array of integers for "start conditions", but there's no adequate documentation for what it explicitly does.

It might be helpful in the future, but I guess it's best to remove it for now?